### ISSUE TITLE
[Bug] Chunk Loaders stop working on server restart

### ISSUE THREAD

**Issue by Jamesthe1** [SuperMartijn642/ChunkLoaders#27]
**Version Info**
- Minecraft 1.12.2
- Chunk Loaders 1.1.7

**Description of the Bug**
We have a server which will be turned off and on at times. When the server is reloaded, chunk loaders stop working, even when you go back to load the chunk they're in.

**Steps to Reproduce**
1. Place any chunk loader in a distant part of the world
2. Restart the server
3. Load the chunk that the chunk loader is in
4. Return to spawn
5. Wait 10-20 seconds
6. Use `/setblock` to try and place a block in one of the chunks you're trying to keep loaded

**Screenshots**
I took note of my exact position in the screenshot, then tried placing a stone block.
![2021-09-24_08 48 40](https://user-images.githubusercontent.com/21296549/134685685-1185d5e4-aee8-49ed-adae-ab3f6596ebd5.png)
![2021-09-24_08 50 35](https://user-images.githubusercontent.com/21296549/134685695-2780082a-6c07-40cb-aef7-6e142cc5b54c.png)

**Comment by DevPGSV** [comment]
I have found the same issue. I am playing in a server on 1.18, using chunkloaders version "chunkloaders-1.1.7-mc1.18".

What we have found on our server is that after the server starts, we have to go to every chunkloader to enable it again, as the configured chunks we have activated in their GUI are all disabled after the restart.

Disabled = every chunk is greyed out and moving the mouse over the chunk displayed "NOT LOADED".
